sips/sip-config-waiting-period.md

```diff
@@ -0,0 +1,84 @@
+---
+sip: 116
+title: Conditionally skipping fee reclamation
+status: Proposed
+author:
+discussions-to:
+created: 2020-02-25
+---
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intends to achieve. This should be non-technical and accessible to a casual community member.-->
+
+Allow waiting period to be set to zero and avoid unecessary fee reclamation functionality.
+
+## Abstract
+
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+
+[SIP-114](./sip-114.md) will enable synth exchanging on L2. Given that transaction (and hence oracle) frontrunning is not possible on Optimisic Ethereum (OE), there is no need for storing exchanges and settling them after a certain amount of time has elapsed. This SIP proposes to set the waiting period to 0 when applicable, as is the case with Optimism.
+
+## Motivation
+
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the SIP proposes changing how something is calculated, you must address *why* the current calculation is innaccurate or wrong. This is not the place to describe how the SIP will address the issue!-->
+
+In Phase 1 of the [transition to Optimistic Ethereum](https://blog.synthetix.io/the-optimistic-ethereum-transition), basic synth functionality is going to be enabled, already proposed in [SIP-114](./sip-114.md). Every time a user exchnages a synth for another, the trade is stored temporarily and can be settled after a certain waiting period (`waitingPeriodSecs` in `SystemSettings`) has elapsed, currently set to 6 minutes. This is mainly done to address oracle front running i.e. monitoring the mempool for oracle price updates and making a favourable trade before the price gets updated. After the aforementioend waiting period has expired, the exchange can be settle on the fair price and the user gets either a rebate or a reclaim. This whole procedure is not needed on OE, thus the waiting period can be set to 0 and skip part of the overhead imposed by settlement.
```

Comment:
`s/exchnges/exchanges`

Reply:
I don't think you need to rehash what fee reclamation is- you can link SIP-37 instead.